internal/config/config.go

```diff
@@ -77,6 +77,9 @@ type Config struct {
 	//   DON'T USE IN PRODUCTION: enable debug mode and use debug images. For usage, see: https://github.com/edgelesssys/constellation/blob/main/debugd/README.md
 	DebugCluster *bool `yaml:"debugCluster" validate:"required"`
 	// description: |
+	//   Attestation variant used to verify the integrity of a node.
+	AttestationVariant string `yaml:"attestationVariant" validate:"valid_attestation_variant"` // TODO: v2.8: Mark required
```

Comment:
(irrespective of the question "should we wait for TDX support to make this required?"): Why start making it required in v2.8 and not v2.7?

Reply:
Because it is not yet required in v2.6 and iirc our "breaking-changes" policy says we want to keep compatibility between two minor version, which we dont do if we start requiring it in v2.7

Reply:
Right. I am a bit confused why I asked the question :D